<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuzzWidget</class>
 <widget class="QWidget" name="FuzzWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Fuzzy HTTP</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="requestVLayout">
     <item>
      <layout class="QGridLayout" name="startingLineGridLayout">
       <item row="2" column="0">
        <widget class="QComboBox" name="requestMethodComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="requestMethodLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Method</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="urlHlLayout">
         <item>
          <widget class="QLabel" name="urlLabel">
           <property name="text">
            <string>URL</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="urlSavedSelectionLabel">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="urlSaveSelectionButton">
           <property name="text">
            <string>Save Selection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="urlLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="requestHeadersLabel">
       <property name="text">
        <string>Headers:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="requestHeadersTableView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="bodyHLayout">
       <item>
        <widget class="QLabel" name="bodyLabel">
         <property name="text">
          <string>Body</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="bodySavedSelectionLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bodySaveSelectionButton">
         <property name="text">
          <string>Save Selection</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="bodyPlainTextEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="payloadVLayout">
     <item>
      <widget class="QLabel" name="urlPayloadLabel">
       <property name="text">
        <string>URL Payload</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="urlButtonsHLayout">
       <item>
        <widget class="QPushButton" name="addUrlPayloadButton">
         <property name="text">
          <string>Add Payload</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteUrlPayloadButton">
         <property name="text">
          <string>Delete Payload</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="urlPayloadTableView">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bodyPayloadLabel">
       <property name="text">
        <string>Body Payload</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="bodyButtonsHLayout">
       <item>
        <widget class="QPushButton" name="addBodyPayloadButton">
         <property name="text">
          <string>Add Payload</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteBodyPayloadButton">
         <property name="text">
          <string>Delete Payload</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="bodyPayloadTableView">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="fuzzVLayout">
     <item>
      <widget class="QLabel" name="sendDelayLabel">
       <property name="text">
        <string>SendDelay</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="sendDelayHlLayout">
       <item>
        <widget class="QSpinBox" name="sendDeaySpinBox">
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="millisecondsLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="replyTableView"/>
     </item>
     <item>
      <widget class="QPushButton" name="fuzzCancelButton">
       <property name="text">
        <string>Fuzz</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
